packages/jsapi-shim/src/PropTypes.js

```diff
@@ -69,7 +69,7 @@ const RollupConfig = PropTypes.shape({
   includeDescriptions: PropTypes.bool,
 });
 
-const User = PropTypes.shape({
+const UserInfo = PropTypes.shape({
```

Comment:
I can't find how `UserInfo` is used. Shouldn't it be the same as `UIPropTypes.User`?

Reply:
No, UserInfo is the object from the API, but it's only in Enterprise anyway so I'll take it out